modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/init.ts

```diff
@@ -130,8 +130,24 @@ export default function init(config) {
 		document.addEventListener('DOMContentLoaded', () => {
 			initSPA(config);
 		});
-	}
-	else {
+	} else {
 		initSPA(config);
 	}
 }
+
+// Properties from `frontend-js-spa-web/init`
+declare global {
+	export interface ILiferay {
+		SPA?: {
```

Comment:
Moved here, this could simply be `SPA?: typeof App`

Reply:
Yeah, I think it would be better because eventually we can modify the App and forget to update the types here.